pkg/ebpf/c/tcp-queue-length-kern.c

```diff
@@ -1,128 +1,101 @@
 #include <linux/kconfig.h>
 #define KBUILD_MODNAME "foo"
-#include <linux/ptrace.h>
 #include <linux/bpf.h>
-#include <net/inet_sock.h>
+#include <linux/ptrace.h>
 #include <linux/tcp.h>
+#include <net/inet_sock.h>
 
 #include "pkg/ebpf/c/bpf-common.h"
 #include "pkg/ebpf/tcp-queue-length-kern-user.h"
 
 /*
- * The `queue` map is used to share with the userland program system-probe
- * the statistics (max/min size of receive/send buffer) of every socket
+ * The `stats` map is used to share with the userland program system-probe
```

Comment:
Can we use a more descriptive name? Perhaps `tcp_queue_stats`?  if we ever want to use this map in another module, the name `stats` won't be very descriptive.

Reply:
Definitively! Change done!